packages/components/src/Popover/Popover.tsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-null/no-null */
```

Comment:
Why is this off? I don't think we should have to disable it.

Reply:
this lint asks us to use undefined instead of null.  We get this strange issue when trying to use undefined with ref

![image](https://user-images.githubusercontent.com/54861184/113731850-192d7580-96c7-11eb-913e-9ba8b38ba9a6.png)

Essentially we're fighting with React's `ref type`.

```
type Ref<T> = { bivarianceHack(instance: T | null): void }["bivarianceHack"] | RefObject<T> | null;
```

the `bivarianceHack(instance: T | null): void }["bivarianceHack"]` is type we're trying to satisfy by passing `setPopperElement` from the `const [popperElement, setPopperElement] = useState<HTMLElement | null>(null);` declaration

If we use undefined, the react typing would complain so hence why we need this disable lint sadly 😢